near-token-factory/src/lib.rs

```diff
@@ -105,9 +147,11 @@ impl Contract {
             .deploy_contract(binary)
             .function_call(
                 "new".to_string(),
-                json!({ "super_admin": None::<AccountId> })
-                    .to_string()
-                    .into_bytes(),
+                near_sdk::serde_json::json!({
+                    "super_admin": self.token_super_admin,
+                })
```

Comment:
`super_admin` parameter of near-token-contract is `Option<AccountId>`:

https://github.com/aurora-is-near/native-erc20-connector/blob/78d504ed2f84120577fa5727c1ace2727b491028/near-token-contract/src/lib.rs#L81

Maybe serde is implicitly doing the conversion? If not, shouldn't passing `AccountId` instead of `Option<AccountId>` cause a test failure?